src/aat/resources/features/F-103/S-603.td.json

```diff
@@ -0,0 +1,36 @@
+{
+  "title": "must return 400 error response for mal-formatted user id list",
+  "_guid_": "S-603",
```

Comment:
guid repeated in multiple files. that will mess things.

Reply:
fixed this!!!! Copy paste error!!

Reply:
Fixed it!!